js/src/get-started-page/features-card/index.js

```diff
@@ -45,27 +90,31 @@ const FeaturesCard = () => {
 						variant="body"
 					>
 						{ __(
-							'Sync your WooCommerce store data with Google Merchant Center to easily list your products across Google.',
+							'When your products are added and approved, they’ll be eligible for free listings, reaching shoppers across Google’s network.',
 							'google-listings-and-ads'
 						) }
 					</Text>
+					<LearnMoreLink
+						linkId="get-started-features-free-listing-learn-more"
+						href="https://woocommerce.com/document/google-listings-and-ads/#google-smart-shopping-campaigns"
```

Comment:
❔ 
The anchor of this link doesn't locate a specific position on the document page. A more relevant link to the content seems https://woocommerce.com/document/google-listings-and-ads/#free-listings-on-google. Is the current one the correct link?

Reply:
Thank you for catching and for the link suggestions. I reviewed the copy against the links and they work. I updated the [marketing copy document](https://docs.google.com/document/d/1TUeMIQakEGzR8Qb45Y-kpVcqIzhKF0bAwb6bccMLWoA/edit?resourcekey=0-06jarg05meLrSMQC1M9N-w#) with the suggested links.

Reply:
Thanks @dominiquevias and @eason9487! Updated in d8c16ba.